prototype/src/main/java/com/google/sps/data/Marker.java

```diff
@@ -0,0 +1 @@
+// Marker.java
```

Comment:
(Nitpick) Perhaps this file could have been added when the saving of the directories was actually implemented, because it is essentially an empty file, and seems to be unrelated to other modified files in this commit.

Reply:
Otherwise LGTM too! 😊

Reply:
True, seperate commits would be better